#!/usr/bin/python3

# raise_exception_msg - raises a name exception with a message
# message: the message to output on NameError
# Return: no return


def raise_exception_msg(message=""):
    raise NameError(message)
